src/utils/mpt/nibbles.cairo

```diff
@@ -0,0 +1,127 @@
+from utils.utils import Helpers
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.math import unsigned_div_rem
+
+struct Nibbles {
+    nibbles_len: felt,
+    nibbles: felt*,
+}
+
+namespace NibblesImpl {
+    func from_bytes{range_check_ptr}(bytes_len: felt, bytes: felt*) -> Nibbles* {
+        alloc_locals;
+        local nibbles_len = bytes_len * 2;
+        let (local output: felt*) = alloc();
+
+        if (nibbles_len == 0) {
+            tempvar res = new Nibbles(nibbles_len, output);
```

Comment:
in done label: `new Nibbles(nibbles_len, nibbles);`
Maybe standardize the naming of what `Nibbles` takes by renaming `output` to `nibbles` or changing the naming in  done label?